scripts/validate

```diff
@@ -11,7 +11,7 @@ echo Running: go vet
 go vet ${PACKAGES}
 
 echo "Running: golangci-lint"
-golangci-lint run --new-from-rev=${headSHA}
```

Comment:
Where is the definition `headSHA`?

Reply:
It looks like golangci-lint allows empty value for `--new-from-rev`. We don't need this anymore, because we fix all golangci-lint issues in this repository.

Reply:
I see. Simply curious about the variable. That's fine. Thank you.